From fb5d7b96f6a57a1d77f46d9bcec2b9c2dd728d65 Mon Sep 17 00:00:00 2001
From: "Kris, Pan" <kris.pan@intel.com>
Date: Tue, 12 Oct 2021 10:23:48 +0800
Subject: [PATCH 096/109] misc: xlink-pcie: remove multipule header file
 mxlk_boot_inf.h

./include/linux/mxlk_boot_inf.h:10:15: error: redefinition of typedef 'mxlk_pcie_boot_event' is a C11 feature [-Werror,-Wtypedef-redefinition]
typedef int (*mxlk_pcie_boot_event)(uint32_t phys_dev_id);
              ^
drivers/misc/xlink-pcie/remote_host/boot.c:11:10: note: './include/linux/mxlk_boot_inf.h' included multiple times, additional include site here
         ^
drivers/misc/xlink-pcie/remote_host/pci.h:15:10: note: './include/linux/mxlk_boot_inf.h' included multiple times, additional include site here
         ^
./include/linux/mxlk_boot_inf.h:10:15: note: unguarded header; consider using #ifdef guards or #pragma once
typedef int (*mxlk_pcie_boot_event)(uint32_t phys_dev_id);

Signed-off-by: Kris, Pan <kris.pan@intel.com>
---
 drivers/misc/xlink-pcie/remote_host/boot.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/misc/xlink-pcie/remote_host/boot.c b/drivers/misc/xlink-pcie/remote_host/boot.c
index 9ba4b4619606..ff203c5c07d2 100644
--- a/drivers/misc/xlink-pcie/remote_host/boot.c
+++ b/drivers/misc/xlink-pcie/remote_host/boot.c
@@ -8,7 +8,6 @@
  ****************************************************************************/
 #include <linux/firmware.h>
 #include <linux/delay.h>
-#include <linux/mxlk_boot_inf.h>
 
 #include "pci.h"
 #include "../common/boot.h"
-- 
2.25.1

